doc/src/devdocs/build/build.md

```diff
@@ -144,7 +144,7 @@ Notes for various architectures:
 Building Julia requires that the following software be installed:
 
 - **[GNU make]**                — building dependencies.
-- **[gcc & g++][gcc]** (>= 5.1) or **[Clang][clang]** (>= 3.5, >= 6.0 for Apple Clang) — compiling and linking C, C++.
+- **[gcc & g++][gcc]** (>= 7.1) or **[Clang][clang]** (>= 5.0, >= 9.3 for Apple Clang) — compiling and linking C, C++.
```

Comment:
Source of this change?

Reply:
For LLVM 15 (couldn't track down for 14 down to 12, all I checked): https://llvm.org/docs/ReleaseNotes.html#update-on-required-toolchains-to-build-llvm

Reply:
So just merge this ("2 failing" must be false alarm)? [Despite not pressing: ] I did want to press "Resolve conversation", but unclear if I should, if then you might not see this.

Reply:
I assume[d] you needed to compile LLVM since "+ [patches]" (and you don't want the system LLVM). If that's wrong, then maybe best you just merge the first commit. It's at least an improvment.

Reply:
You don't need to compile LLVM since an already compiled version is downloaded. You can remove the last commit and this can be merged.